app/src/main/kotlin/com/wire/android/ui/home/conversationslist/common/ConversationItemFactory.kt

```diff
@@ -108,6 +116,9 @@ fun ConversationItemFactory(
 private fun GeneralConversationItem(
     searchQuery: String,
     conversation: ConversationItem,
+    belongsToGroup: Boolean,
```

Comment:
also not clear to me what this means, maybe a simple checked would be better here ? or maybe create a custom data object like Clickable that can be passed to GeneralConversationItem with all the three variables belongsToGroup, selectonRadioGrouop and isSelectable